### PR TITLE
Repository improvements

### LINKED ISSUE #1037: Wazuh QA repository enhancements
This repository must contain the following elements:
- A changelog file to log any update.
- A version file.
- Update the README.md content.
- Remove deprecated directories (`jenkins`, `ansible` and `kitchen`)

### PR DESCRIPTION
**Pull request by BraulioV**
|Related issue|
|---|
|closes #1037 |

## Description

This PR adds all the missing components to the repository and listed in #1037 as well as new PR templates and updated copyright dates.

## Configuration options

NA

## Logs example

NA

## Tests

Not necessary.

### CODE REVIEW

**Review by jmv74211 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -1,58 +1,38 @@
...
 
-`<YOUR_INTRODUCTION_TEXT>`
-
-Closes #`<YOUR_ISSUE_NUMBER>`
-
-# Tests logic
+<!--
+This template reflects sections that must be included in new Pull requests.
+Contributions from the community are really appreciated. If this is the case, please add the
+"contribution" to properly track the Pull Request.
```
Comment (SpotHound) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:8]: the "contribution" label?

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -1,58 +1,38 @@
...
+## Logs example
 
+<!--
+Paste here related logs and alerts
+-->
 
-```
-<PYTEST_RESULT>
-```
+## Tests
```
Comment (SpotHound) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:31]: We could add a step to add a screenshot of the execution of the test (?)

**Review by SpotHound — APPROVED** [review]
LGTM only I add a few suggestions.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 2 above)
Comment (BraulioV) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:31]: I think the _Logs example_ section covers that

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 1 above)
Comment (BraulioV) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:8]: This [one](https://github.com/wazuh/wazuh-qa/labels/community).

**Pull request merged**